lib/vspc/vspc.go

```diff
@@ -179,7 +187,9 @@ func (vspc *Vspc) Start() {
 func (vspc *Vspc) Stop() {
 	defer trace.End(trace.Begin("stop vspc"))
 
-	vspc.doneCh <- true
+	vspc.Server.Close()
```

Comment:
I think if vspc.Server.Close() closes acceptance of incoming connections and actually interrupt Accept() call, closing vspc.doneCh first would make more sense.